docs/technical_reference/flowsheets/generic_desalination_train.rst

```diff
@@ -0,0 +1,105 @@
+            * m.fs.{unit_name}.desalter.brine_solids_concentration (kg/m^3) 
+            * m.fs.{unit_name}.desalter.brine_water_mass_percent (%) 
+    * Desalination unit operation (Fix only one)
+        * Water recovery (m.fs.{unit_name}.desalter.water_recovery (in %))
+        * (or) Brine solids concentration (m.fs.{unit_name}.desalter.water_recovery (in %))
+        * (or) Brine water content (m.fs.{unit_name}.desalter.water_recovery (in %))
+    * Valorization unit costs: 
+        * Base cost (m.fs.Pretreatment.costing.base_cost in $/m^3 of feed) 
+        * (Optional) Target product removal value (m.fs.Valorizer.separator.product_value[{component}] in % (Default: 0 for all components))
+        * (Optional) Chemical cost (m.fs.Valorizer.separator.chemical_cost in $/kg (Default: 0)
+    * Valorization unit operation: 
+        * (Optional) Target product removal amount (m.fs.Valorizer.separator.component_removal_percent[{component] in % (Default: 0 for all components))
+        * (Optional) Chemical addition amount (m.fs.Valorizer.separator.additive_cost in PPM if any (Default: 0) 
+
+
+The flowsheet is populated with baseline costs for the unit operations based on general literature values, as presented in Figure 1. 
+
+
+Flowsheet Specifications
+------------------------
+
+These are the default values configured in flowsheet main function: 
+
+.. csv-table::
+   :header: "Description", "Value", "Units"
+
+   "**Feed Water**"
+   "Water mass flow","1", ":math:`\text{kg/s}`"
+   "TDS concentration", "3.5", ":math:`\text{g/L}`"
+   "ion X", "0.3", ":math:`\text{g/L}`"
+   "Temperature", "298.15", ":math:`\text{K}`"
+   "Pressure", "101325", ":math:`\text{Pa}`"
+
+   "**Stream costing**"
+   "Product distribution", "0.0", ":math:`\text{$/}\text{m}^3`"
+   "Feed sourcing", "0.0", ":math:`\text{$/}\text{m}^3`"
+   "Waste disposal", "1.0", ":math:`\text{$/}\text{m}^3`"
+   "Additive cost", "0.0", ":math:`\text{$/kg}`"
+   "Additive dose", "1.0", ":math:`\text{PPM}`"
+   "Component removal percent [X]", "50", ":math:`\text{%}`"
+   "Separation cost [X]", "0.5", ":math:`\text{$/kg}`"
+
+   "**Desal_1**"
+   "Base cost", "0.3", ":math:`\text{$/}\text{m}^3`"
+   "Recovery", "80", ":math:`\text{%}`"
+
+   "**Desal_2**"
+   "Base cost", "0.5", ":math:`\text{$/}\text{m}^3`"
+   "Recovery cost", "0.01", ":math:`\text{$/}(\text{\%wr*m}^3)`"
+   "Recovery cost offset", "35", ":math:`\text{\%}`"
+   "Recovery", "50", ":math:`\text{\%}`"
+
+   "**Desal_3**"
+   "Base cost", "10", ":math:`\text{$/m^3}`"
+   "Brine water mass percent", "80", ":math:`\text{\%}`"
+
+   "**Valorizer**"
+   "Base cost", "0", ":math:`\text{$/}\text{m}^3`"
+   "Additive dose", "0", ":math:`\text{PPM}`"
+   "Additive cost", "0", ":math:`\text{$/}\text{m}^3`"
+   "Component removal percent [X]", "50", ":math:`\text{\%}`"
+   "Product value [X]", "1", ":math:`\text{$/kg}`"
```

Comment:
```suggestion
   "**Feed Water**"
   "Water mass flow","1", ":math:`\text{kg/s}`"
   "TDS concentration", "3.5", ":math:`\text{g/L}`"
   "ion X", "0.3", ":math:`\text{g/L}`"
   "Temperature", "298.15", ":math:`\text{K}`"
   "Pressure", "101325", ":math:`\text{Pa}`"

   "**Stream costing**"
   "Product distribution", "0.0", ":math:`\text{\$/}\text{m}^3`"
   "Feed sourcing", "0.0", ":math:`\text{\$/}\text{m}^3`"
   "Waste disposal", "1.0", ":math:`\text{\$/}\text{m}^3`"
   "Additive cost", "0.0", ":math:`\text{\$/kg}`"
   "Additive dose", "1.0", ":math:`\text{PPM}`"
   "Component removal percent [X]", "50", ":math:`\text{%}`"
   "Separation cost [X]", "0.5", ":math:`\text{\$/kg}`"

   "**Desal_1**"
   "Base cost", "0.3", ":math:`\text{$/}\text{m}^3`"
   "Recovery", "80", ":math:`\text{%}`"

   "**Desal_2**"
   "Base cost", "0.5", ":math:`\text{\$/}\text{m}^3`"
   "Recovery cost", "0.01", ":math:`\text{\$/}(\text{\%wr*m}^3)`"
   "Recovery cost offset", "35", ":math:`\text{%}`"
   "Recovery", "50", ":math:`\text{%}`"

   "**Desal_3**"
   "Base cost", "10", ":math:`\text{\$/m^3}`"
   "Brine water mass percent", "80", ":math:`\text{%}`"

   "**Valorizer**"
   "Base cost", "0", ":math:`\text{\$}\text{m}^3`"
   "Additive dose", "0", ":math:`\text{PPM}`"
   "Additive cost", "0", ":math:`\text{\$/}\text{m}^3`"
   "Component removal percent [X]", "50", ":math:`\text{%}`"
   "Product value [X]", "1", ":math:`\text{\$/kg}`"
```

Reply:
I'll try this change next if what I just did doesn't work...